### ISSUE TITLE
[REQUEST] Display currently installed release iteration

### ISSUE THREAD

**Issue by MisterChocker** [flybywiresim/installer#61]
**Describe the feature you'd like to see implemented**

I'd like to see/know my currently installed release as well. like the release number e.g. 0.5.2
![image](https://user-images.githubusercontent.com/1857322/104140120-0b11d680-53b0-11eb-8fc3-d3c22fe4f00a.png)

**Additional context**

Discord username (if different from GitHub): DerL30N#3751

**Comment by Benjozork** [comment]
this text appears when you select a new version like dev, stable or cfbw. are you sure you are not confusing this with releases like 0.5.0 ?

**Comment by ZigTag** [comment]
I think they mean they want to see what version they currently have installed ex v0.5.2, (checksum), etc

**Comment by MisterChocker** [comment]
Yes, I'd like to know what version I am currently using/running/have installed.

**Comment by Benjozork** [comment]
Clarification

Version = stable/dev/fbw

Release = 0.5.2/0.5.1

Which one are you referring to ?

ps: we should probably change this naming

**Comment by MisterChocker** [comment]
@Benjozork release

**Comment by nistei** [comment]
@ZigTag why did this get closed?

**Comment by ZigTag** [comment]
Oops your right I thought it meant installer version
